src/App.js

```diff
@@ -0,0 +1,74 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import { AiFillPlusCircle } from "react-icons/ai";
+import TaskList from "./components/ui/TaskList";
+const App = () => {
+  const [task, setTask] = useState("");
+  const [toDo, setAllToDo] = useState([]);
+
+  const handleOnChange = (event) => {
+    setTask(event.target.value);
+  };
+  const handleTasksubmit = (event) => {
+    if (task !== "") {
+      const newTask = { id: nanoid(), taskName: task, isTaskDone: false };
+      event.preventDefault();
+
+      setAllToDo((prevToDoList) => [...prevToDoList, newTask]);
+      setTask("");
+    }
+  };
+
+  const handleTaskDone = (id) => {
+    console.log(id);
```

Comment:
remove this console.log, there is no need to have them in the code, as soon as you're done debugging you can remove them!